Makefile

```diff
@@ -86,7 +86,7 @@ update-gce-images:
 	bin/gce-image-update $$(git grep -lE 'source_image: ubuntu' *.yml)
 
 tmp/packer.zip:
-	$(CURL) -sSLo $@ 'https://releases.hashicorp.com/packer/1.0.0/packer_1.0.0_$(UNAME)_amd64.zip'
+	$(CURL) -sSLo $@ 'https://releases.hashicorp.com/packer/1.0.4/packer_1.0.4_$(UNAME)_amd64.zip'
```

Comment:
@meatballhat I noticed in a commit we bumped to v1.0.4.  Any eason why not v1.1.0?  If I remember correctly, running v1.0.4 locally introduced problems with other builder types.

Reply:
I'm good with a bump to v1.1.0 👍